docs/prover/cli/options.md

```diff
@@ -793,29 +793,117 @@ Advanced options that control control flow splitting
 
 See [here](control-flow-splitting) for an explanation of control flow splitting.
 
+(-depth)=
+### `--prover_args '-depth <number>'`
+
+**What does it do?**
+
+Sets the maximum splitting depth.
+
+**When to use it?** 
+
+When the deepest splits are too heavy to solve, but not too high in number,
+increasing this will lead to smaller, but more split leafs, which run at the
```

Comment:
```suggestion
increasing this will lead to smaller, but more split leaves, which run at the
```

Reply:
uh, I knew that at some point .. in high school .. changing